packages/storybook/stories/additional-docs.js

```diff
@@ -5,31 +5,56 @@ const { AVAILABLE, BEST_PRACTICE, CANDIDATE, DEPLOYED, DEPRECATED, PROPOSED } =
   level;
 
 export const additionalDocs = {
+  'Accept terms prompt': {
+    maturityCategory: CAUTION,
+    maturityLevel: AVAILABLE,
+  },
+  'Accordion': {
+    maturityCategory: USE,
+  },
   // MDX
   'Action link': {
     guidanceHref: 'link/action',
     guidanceName: 'Action link',
     maturityCategory: USE,
     maturityLevel: DEPLOYED,
   },
+  'Additional info': {
+    maturityCategory: USE,
+  },
   // MDX
   'Address block': {
     guidanceHref: 'address-block',
     guidanceName: 'Address block',
     maturityCategory: USE,
     maturityLevel: DEPLOYED,
   },
+  'Alert': {
+    maturityCategory: USE,
+  },
+  'Alert - expandable': {
+    maturityCategory: CAUTION,
+  },
+  'Back to top': {
+    maturityCategory: USE,
+  },
+  'Banner': {
+    maturityCategory: USE,
+  },
+  'Breadcrumbs': {
+    maturityCategory: USE,
+  },
```

Comment:
Is there a reason why these don't have a `maturityLevel`?

Reply:
Those are only used for the sidebar indicators, so a `maturityLevel` is not needed.